build-logic/convention/src/main/kotlin/com/google/samples/apps/nowinandroid/AndroidCompose.kt

```diff
@@ -38,5 +38,33 @@ internal fun Project.configureAndroidCompose(
         composeOptions {
             kotlinCompilerExtensionVersion = libs.findVersion("androidxCompose").get().toString()
         }
+
+        kotlinOptions {
+            freeCompilerArgs = freeCompilerArgs + buildComposeMetricsParameters()
+        }
+    }
+}
+
+private fun Project.buildComposeMetricsParameters(): List<String> {
+    val metricParameters = mutableListOf<String>()
```

Comment:
I think buildList{} could be used, but that API is still ExperimentalStdlibApi, so I am using MutableList.

Reply:
Yup that makes sense, `mutableListOf` looks fine!